src/AppBundle/EventListener/CheckSQLiteEventSubscriber.php

```diff
@@ -69,7 +70,12 @@ public function handleDatabaseExceptions(ConsoleExceptionEvent $event)
      */
     public function onKernelException(GetResponseForExceptionEvent $event)
     {
-        if (!extension_loaded('sqlite3')) {
+        $exception = $event->getException();
+        // Since any exception thrown during a Twig template rendering is wrapped in a Twig_Error_Runtime.
+        // We must get the original exception.
+        $previousException = $exception->getPrevious();
```

Comment:
can be in one line `$previousException = $event->getException()->getPrevious();`